plugins/com.google.cloud.tools.eclipse.appengine.facets/plugin.properties

```diff
@@ -2,5 +2,8 @@ google = Google
 Bundle-Vendor = Google Inc.
 Bundle-Name = Facets for Google Cloud Platform Apps
 
-facetName = Google App Engine Standard Java
-facetDescription = Supply App Engine Functionality
+flexFacetName = Google App Engine Flex Java
+flexFacetDescription = Supply App Engine Flex Functionality
```

Comment:
I don't think so. 

This naming seems off to me. 

II would remove the 'google' and call these facets 'App Engine Standard' and 'App Engine Flexible Environment'.

@arouzrokh should review this.